openedx/adg/admin_dashboard/admin_settings/forms.py

```diff
@@ -0,0 +1,66 @@
+"""
+Forms for `admin_settings` app
+"""
+from django import forms
+from django.contrib.auth.models import User
+
+from openedx.adg.admin_dashboard.admin_settings.constants import (
+    ALREADY_ADMIN_ERROR,
+    ALREADY_INVITED_ADMIN_ERROR,
+    BUSINESS_UNIT,
+    BUSINESS_UNIT_REQUIRED_ERROR,
+    PERMISSION
+)
+from openedx.adg.admin_dashboard.admin_settings.models import Invitation
+from openedx.adg.lms.applications.constants import EMAIL
+from openedx.adg.lms.applications.helpers import has_admin_permissions
+
+
+class InvitationForm(forms.ModelForm):
+    """
+    Invitation form for team member invitation
+    """
+
+    class Meta:
+        model = Invitation
+        fields = ['email', 'business_unit', 'permission', 'status']
+
+    def clean(self):
+        """
+        Raise error if there's already pending or accepted invitation against this email or a user is registered using
+        this email as an admin.
+        """
+        cleaned_data = super().clean()
+        self.validate_email(cleaned_data)
+        self.validate_business_unit(cleaned_data)
```

Comment:
We should not change the return type of a method that we are overriding, this should return cleaned_data at the end.